src/LinuxTracing/PerfEvent.h

```diff
@@ -181,6 +181,8 @@ using GenericTracepointPerfEvent = TypedPerfEvent<GenericTracepointPerfEventData
 struct TaskNewtaskPerfEventData {
   char comm[16];
   pid_t new_tid;
+  pid_t was_blocked_by_thread;
+  pid_t was_blocked_by_process;  // Could be zero if we don't have access to the process id.
```

Comment:
Again, I need clarification on "Could be zero if we don't have access to the process id".